src/v2/view-builder/views/idp/EnrollAuthenticatorIdPBody.js

```diff
@@ -0,0 +1,23 @@
+import { loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+
+const Body = BaseForm.extend({
```

Comment:
I don't see any visual difference between this and `ChallengeAuthenticatorIdPBody.js` view.. what i am missing ? Can't we use the save view ?

Reply:
The primary difference between the two views are the labels used for the title, subtitle, and save action.

Reply:
those can be passed as option to the view ?

Reply:
Can you give an example of how we can conditionally do this? I wasn't able to find an example of us doing something like the previously, and figured this was the cleanest solution. 😄

Reply:
When you are adding the this view just past the correct options  ?

Reply:
Figured out what you were trying to say here -

Essentially, we can initialize a set of `options` that our child `Body` can use within our `AuthenticatorIdPView` object. In turn, these can be used to conditionally render our labels (`title`, `subtitle`, and `save`).

Updated the PR to do this, as it removes the need to maintain an Enroll/Challenge view!

Reply:
You could also do what I did for Duo, which is have an IdPBaseForm and reuse it in Challenge and Enroll views, extending to have the correct title and the correct Footer

Reply:
⬆️ This is what I ended up doing - keeping all logic within a single view. Thanks for the suggestions all!